cmd/algorand-indexer/main.go

```diff
@@ -218,8 +194,11 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Setup our exit handler for maybeFail() and other exit panics
+	defer exitHandler()
+
 	if err := rootCmd.Execute(); err != nil {
 		logger.WithError(err).Error("an error occurred running indexer")
-		os.Exit(1)
+		panic(exit{1})
```

Comment:
If we get here any defers inside the execute function would have already been processed, so no need to use the panic trick.